README.md

```diff
@@ -147,6 +147,13 @@ Returns a thunk which when executed deletes the entire database.
 yield db.drop();
 ```
 
+#### list
+Returns a thunk which when executed list existing collections in the database.
+
+```js
+yield db.list();
```

Comment:
`collectionNames` exists on 1.4 driver, but is deprecated in 2.0; `collections` exists on 2.0 driver but does different things.

just going back to `listCollections` is probably safer.